pymatviz/powerups/__init__.py

```diff
@@ -2,6 +2,8 @@
 matplotlib and plotly figures.
 """
 
+from __future__ import annotations
```

Comment:
`pymatviz` is 3.10+. originally, 3.9 was meant to be the last release that needed `from __future__ import annotations` for future type hints to work. i think that was pushed back to 3.12. still not clear to me that we use any annotation features that don't work without it on 3.10 and above. given CI runs fine on 3.10, i think we don't need these

Reply:
> originally, 3.9 was meant to be the last release that needed from __future__ import annotations for future type hints to work. i think that was pushed back to 3.12.

[It seems the postponed evaluation is still in TBD state](https://docs.python.org/3/library/__future__.html).

> from __future__ import annotations was previously scheduled to become mandatory in Python 3.10, but the Python Steering Council twice decided to delay the change ([announcement for Python 3.10](https://mail.python.org/archives/list/python-dev@python.org/message/CLVXXPQ2T2LQ5MP2Y53VVQFCXYWQJHKZ/); [announcement for Python 3.11](https://mail.python.org/archives/list/python-dev@python.org/message/VIZEBX5EYMSYIJNDBF6DMUMZOCWHARSO/)). No final decision has been made yet. See also [PEP 563](https://peps.python.org/pep-0563/) and [PEP 649](https://peps.python.org/pep-0649/).

That caused some failure in this PR (I cannot remember which module, I guess its `rdf` in cf8eb60e087b713a96f9782ccec44baf5fb08549), and then I added them for all modules just in case.

Reply:
looks like linting and tests on https://github.com/janosh/pymatviz/commit/cf8eb60e087b713a96f9782ccec44baf5fb08549 passed? can we try removing the future imports to see if/where it fails?

Reply:
so looks like we do need to the future annotations, at least in `rdf.py`?

Reply:
I believe we still need postponed evaluation for most (if not all) of our code, otherwise all type annotations need to be resolved at runtime (i.e. we cannot use `if TYPE_CHECKING` at all)

Reply:
ok, good to know. let's keep it then